src/renderer/before-frame-starts/tokens.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectionToken } from "@ogre-tools/injectable";
+import type { Runnable } from "../../common/runnable/run-many-for";
+
+export const evenBeforeFrameStartsInjectionToken = getInjectionToken<Runnable>({
+  id: "even-before-frame-starts",
+});
+
+// NOTE: these are only run when process.isMainFrame === true
+export const evenBeforeMainFrameStartsInjectionToken = getInjectionToken<Runnable>({
+  id: "even-before-main-frame-starts",
+});
+
+// NOTE: these are only run when process.isMainFrame === false
+export const evenBeforeClusterFrameStartsInjectionToken = getInjectionToken<Runnable>({
+  id: "even-before-cluster-frame-starts",
+});
```

Comment:
Since all these tokens located in folder `before-frame-starts` shouldn't we have a convention about their names starting from `beforeFrameStarts[a-Z]+` for example? _(like below one)._ 
Also as benefit would be easier to (auto)import these tokens with IDE by start typing known prefixed names.

Reply:
Good point. Will try and make better names

Reply:
How about now?

Reply:
Looking good.